### PR TITLE
Upgrade pregen command

### PR DESCRIPTION
**Pull request by svdgoor**
PregenTask now takes width & height ins o radius
Split pregen command into 
- /iris pregen create <stuff>
- /iris pregen stop
- /iris pregen toggle
- /iris pregen
I'll be wrapping up the WIP parts (descriptions, arg usages etc) & wiki

### CODE REVIEW

File: src/main/java/com/volmit/iris/core/command/pregen/CommandIrisPregenStop.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @Override
+    public void addTabOptions(VolmitSender sender, String[] args, KList<String> list) {
+
+    }
+
+    @Override
+    protected String getArgsUsage() {
+        return null;
```
Comment (StrangeOne101) [inline src/main/java/com/volmit/iris/core/command/pregen/CommandIrisPregenStop.java]: Maybe return "" instead

**Review by StrangeOne101 — APPROVED** [review]
Looks good 👍

File: src/main/java/com/volmit/iris/core/command/pregen/CommandIrisPregenStop.java (hunk 1 above)
Comment (svdgoor) [reply src/main/java/com/volmit/iris/core/command/pregen/CommandIrisPregenStop.java]: It's a WIP

**Comment by svdgoor** [comment]
Works like a charm
image
image

Only problem is that the GUI is incapable of showing off-center / scaled (? idk) windows 
image

**Comment by cyberpwnn** [comment]
Why are there two PRs for the same thing?

**Comment by cyberpwnn** [comment]
Oh... I see, ill close this one, refer to #506 where I reviewed changes.

**Pull request closed without merging**